samples/display/lvgl/src/main.c

```diff
@@ -30,7 +31,10 @@ void main(void)
 		return;
 	}
 
-	hello_world_label = lv_label_create(lv_scr_act(), NULL);
+	hello_world_button = lv_btn_create(lv_scr_act(), NULL);
```

Comment:
Let me test it on different displays, probably button could be conditional.

Reply:
The button does not look good on monochrome displays because of the background gradient, I suggest to use the button only if KSCAN is enabled, e.g.:

```diff
diff --git a/samples/display/lvgl/src/main.c b/samples/display/lvgl/src/main.c
index c40808258d..c566b52922 100644
--- a/samples/display/lvgl/src/main.c
+++ b/samples/display/lvgl/src/main.c
@@ -31,10 +31,14 @@ void main(void)
                return;
        }
 
-       hello_world_button = lv_btn_create(lv_scr_act(), NULL);
-       lv_obj_align(hello_world_button, NULL, LV_ALIGN_CENTER, 0, 0);
+       if (IS_ENABLED(CONFIG_KSCAN)) {
+               hello_world_button = lv_btn_create(lv_scr_act(), NULL);
+               lv_obj_align(hello_world_button, NULL, LV_ALIGN_CENTER, 0, 0);
+               hello_world_label = lv_label_create(hello_world_button, NULL);
+       } else {
+               hello_world_label = lv_label_create(lv_scr_act(), NULL);
+       }
 
-       hello_world_label = lv_label_create(hello_world_button, NULL);
        lv_label_set_text(hello_world_label, "Hello world!");
        lv_obj_align(hello_world_label, NULL, LV_ALIGN_CENTER, 0, 0);

```

Reply:
It's a bit awkward to conditionalize the sample like this. What if we instead configured monochrome displays to use the [mono theme](https://littlevgl.com/docs/themes/lv_theme_mono_1.png) by default?

Reply:
> It's a bit awkward to conditionalize the sample like this.

no processor macros are awkward, `if (IS_ENABLED(CONFIG_OPTION)) {` is fine.

>What if we instead configured monochrome displays to use the [mono theme](https://littlevgl.com/docs/themes/lv_theme_mono_1.png) by default?

yes, an option. But it looks anyway ugly on small displays, the button size does not scale, probably `lv_btn_set_fit` would help.

Reply:
`lv_btn_set_fit(hello_world_button, LV_FIT_TIGHT);` did not work well, the button is still too big for a 128x64 pixel display.

Reply:
I'm not fond of making the button conditional in a simple "hello world" type of example, but I don't have a 128x64 pixel display to play with making the button fit. I updated the patch to your suggestion.